### PR TITLE
chore(fix): fixing recover account and minor changes

### LINKED ISSUE #4759: Recover Account needs to be restored
We need to allow users to restore their accounts with the seed. Right now this option is disabled. Improvements can come later

### PR DESCRIPTION
**Pull request by JustZacca**
### What this PR does 📖
Self-explanatory: This PR fixes the recover account function and does some other minor changes.

### Which issue(s) this PR fixes 🔨
- Resolve #4759

### CODE REVIEW

File: libraries/Phantom/PhantomManager.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default class PhantomManager {
    **/
   async initWallet() {
     if (this.provider.isPhantom) {
+      await this.$PhantomWalletAdapter.connect()
```
Comment (stavares843) [inline libraries/Phantom/PhantomManager.ts:29]: do we need this duplicated line of code?

File: store/accounts/actions.ts [hunk 2]
```diff
@@ -360,10 +364,10 @@ export default {
...
 
     const { pinHash } = state
-    const entropyMessage = IdentityManager.generateEntropyMessage(
-      $BlockchainClient.account.publicKey.toBase58(),
-      pinHash,
-    )
+    // const entropyMessage = IdentityManager.generateEntropyMessage(
+    //   $BlockchainClient.account.publicKey.toBase58(),
+    //   pinHash,
+    // )
```
Comment (stavares843) [inline store/accounts/actions.ts:370]: do we need this commented out code?

**Review by stavares843 — CHANGES_REQUESTED** [review]
image


when you import a account asks you the username again


also, not as relevant but its cut off in the top when importing the account

image

File: store/accounts/actions.ts (hunk 2 above)
Comment (JustZacca) [reply store/accounts/actions.ts:370]: Same as before

**Comment by JustZacca** [comment]
Maybe is a configuration of your iridium node, because for me it works without re asking for the username.
It's a strange topic, maybe if @WanderingHogan can test it to verify

**Comment by stavares843** [comment]
my iridium node? im using the fleek URL

hogan has better things to do than going to test something that i already did @JustZacca

File: libraries/Phantom/PhantomManager.ts (hunk 1 above)
Comment (JustZacca) [reply libraries/Phantom/PhantomManager.ts]: removed
I'll wait for the proper fix on Multi Wallet

**Comment by JustZacca** [comment]
Okay, talking with @molimauro I've discovered an issue with the remote nodes of Iridium. We thought that it would be better to wait for the merge till it gets fixed.

**Review by aewing — APPROVED** [review]

**Comment by josephmcg** [comment]
I could not successfully recover my account. it pushed me to new registration then was spinning forever

**Comment by JustZacca** [comment]
> I could not successfully recover my account. it pushed me to new registration then was spinning forever

Okay, then is not an Iridium problem, the account recovery is inconsistent (sometimes it happens, sometimes it does not) I will investigate that

It could be still an Iridium Problem (not a version problem but a settings problem) so idk
I'll investigate

**Pull request closed without merging**